packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -3934,7 +3934,7 @@ export class ThinEngine {
                     texture.onLoadedObservable.remove(onLoadObserver);
                 }
 
-                if (EngineStore.UseFallbackTexture) {
+                if (EngineStore.UseFallbackTexture && loadFallback) {

```

Comment:
Why not in this case always place UseFallbackTexture to false instead of adding an extra param ?

Reply:
This would affect all errors, not only the CSP ones so I didn't want to change it

Reply:
But in case of CSP error, at least once, the fallback would anyway not work for any errors ?